semgrep_output_v1.atd

```diff
@@ -1113,29 +1114,35 @@ type scan_request = {
 }
 
 (* Response by the backend to the CLI to the POST /scans *)
-(* TODO
 type scan_response = {
-    scan: scan_response_bis;
+    scan: scan_info;
+    scan_config: scan_configuration;
+    engine_config: engine_configuration;
 }
-type scan_response_bis = {
-    (* the scan id *)
-    id: string;
-    (* since 1.37.0 *)
-    ?enabled_products: product list;
+
+(* meta info about the scan *)
+type scan_info = {
+    id: int; (* the scan id *)
+    deployment_id: int;
+    deployment_name: string;
+}
+
+(* config specific to the scan, eg *)
+type scan_configuration = {
+    enabled_products: product list;
```

Comment:
But this will break old CLI right? They are expecting this enabled_products thing
under scan:, not under scan_config: in the scan_response.